src/Files.App/Filesystem/Search/FolderSearch.cs

```diff
@@ -1,65 +1,60 @@
 // Copyright (c) 2023 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
-using CommunityToolkit.Mvvm.DependencyInjection;
-using CommunityToolkit.WinUI;
-using Files.App.Data.Items;
-using Files.App.Extensions;
 using Files.App.Filesystem.StorageItems;
-using Files.App.Helpers;
-using Files.App.ViewModels;
-using Files.Backend.Services.Settings;
-using Files.Shared.Extensions;
+using Files.Backend.Helpers;
 using Microsoft.Extensions.Logging;
-using System;
-using System.Collections.Generic;
-using System.Collections.ObjectModel;
 using System.IO;
-using System.Linq;
-using System.Threading;
-using System.Threading.Tasks;
 using Windows.Storage;
 using Windows.Storage.FileProperties;
 using Windows.Storage.Search;
-using static Files.Backend.Helpers.NativeFindStorageItemHelper;
-using FileAttributes = System.IO.FileAttributes;
-using WIN32_FIND_DATA = Files.Backend.Helpers.NativeFindStorageItemHelper.WIN32_FIND_DATA;
 
 namespace Files.App.Filesystem.Search
 {
+	/// <summary>
+	///  Provides an utility for storage folder search.
+	/// </summary>
 	public class FolderSearch
 	{
-		private IUserSettingsService UserSettingsService { get; } = Ioc.Default.GetRequiredService<IUserSettingsService>();
-		private DrivesViewModel drivesViewModel = Ioc.Default.GetRequiredService<DrivesViewModel>();
+		private const uint DEFAULT_STEP_SIZE = 500;
 
-		private readonly IFileTagsSettingsService fileTagsSettingsService = Ioc.Default.GetRequiredService<IFileTagsSettingsService>();
+		private readonly IUserSettingsService _userSettingsService;
 
-		private const uint defaultStepSize = 500;
+		private readonly DrivesViewModel _drivesViewModel;
+
+		private readonly IFileTagsSettingsService _fileTagsSettingsService;
```

Comment:
Can you move this to line 22 so the services are together?

Reply:
Sure.